src/solvers/dgsem_tree/dg_1d_parabolic.jl

```diff
@@ -147,11 +146,11 @@ function calc_volume_integral!(du, flux_viscous,
 end
 
 # This is the version used when calculating the divergence of the viscous fluxes
-function prolong2interfaces!(cache_parabolic, flux_viscous,
+function prolong2interfaces!(cache, flux_viscous::Array{uEltype, 3},
```

Comment:
See above

Reply:
So here we indeed need this since `cache_parabolic` is no longer passed around.

We need to distinguish between

https://github.com/trixi-framework/Trixi.jl/blob/9a1e585ea89b1a2f0eb1beaa7d90d3cabf930cfc/src/solvers/dgsem_tree/dg_1d.jl#L408-L425

and 

https://github.com/trixi-framework/Trixi.jl/blob/9a1e585ea89b1a2f0eb1beaa7d90d3cabf930cfc/src/solvers/dgsem_tree/dg_1d_parabolic.jl#L149-L171

Reply:
Actually for 1D this could be removed. I added comments for the other routines, though.